bark_ml/library_wrappers/lib_fqf_iqn_qrdqn/model_wrapper/pytorch_script_wrapper.cpp

```diff
@@ -0,0 +1,16 @@
+#include "bark_ml/library_wrappers/lib_fqf_iqn_qrdqn/model_wrapper/model_loader.hpp"
```

Comment:
make this additional as in BARK

Reply:
I am not sure if i got you correct. Do you mean, separate it from barkml and create a separate bazel project, and reference it from barkml?

Reply:
No, I just don't want to build pytorch every single time. I guess there are tests that require that? Have a look in bark/models/behavior/planners. That's what I meant

Reply:
We do not have to build pytorch, the shared libraries are prebuilt in the pip-package. So this is not more overhead than building some cpp observer.